src/aap_eda/core/validators.py

```diff
@@ -58,6 +54,19 @@ def check_if_de_exists(decision_environment_id: int) -> int:
     return decision_environment_id
 
 
+def check_if_de_valid(image_url: str, eda_credential_id: int):
+    credential = get_credential_if_exists(eda_credential_id)
+    inputs = yaml.safe_load(credential.inputs.get_secret_value())
+    host = inputs.get("host")
+
+    if host and not image_url.startswith(host):
```

Comment:
Raise an exception if host is empty

Reply:
@benthomasson Host is a required parameter in the object https://github.com/ansible/eda-server/blob/a7ffc3397288f944088d6bbf44dfb9a72a7cb7bb/src/aap_eda/core/management/commands/create_initial_data.py#L210 Without host we won't be able to create a registry credential in the DB

Reply:
I'd still like this to be enforced here in the check.

Reply:
Ok, I'll add it

Reply:
@benthomasson the raising exception is added when host is empty